demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -33,7 +34,7 @@ def build_argparser():
     parser = ArgumentParser(add_help=False)
     args = parser.add_argument_group('Options')
     args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
-    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model",
+    args.add_argument("-m", "--model_path", help="Required. Path to an .xml file with a trained model",
```

Comment:
why do you need to change cmd parameter name?

Reply:
if it is really required, then you need to update parameters in demo readme

Reply:
It was just for the naming.
I will reverse these changes.

Reply:
Done